src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -242,7 +242,7 @@ public override AssemblyName GetName(bool copiedName)
             return type;
         }
 
-        [RequiresUnreferencedCode("Types might be removed")]
+        [RequiresUnreferencedCode("Types might be removed by trimming. Consider using Type.GetType instead.")]
```

Comment:
If we're updating CoreCLR's RuntimeAssembly, should we also update Mono's and NativeAOT's?

Reply:
Done - thanks for reminding me